src/com/pugh/sockso/web/action/api/BaseApiAction.java

```
@@ -3,12 +3,39 @@
 
 import com.pugh.sockso.web.action.BaseAction;
 
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 abstract public class BaseApiAction extends BaseAction implements ApiAction {
 
     public static final int DEFAULT_OFFSET = 0;
     
     public static final int DEFAULT_LIMIT = 100;
     
+    public static final String FROM_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
+
+    @Override
+    public Date getFromDate() {
+
+        Date dateFrom = null;
+
+        try {
```

I think we should throw this exception instead of catching it.  If it's caught like this the caller doesn't get notified if using incorrect formats (could be a nightmare to debug).